infra/aws/terraform/prow-build-cluster/terraform.canary.tfvars

```
@@ -19,28 +19,19 @@ eks_admins = [
   "xmudrii"
 ]
 
-assume_role                 = true
-deploy_kubernetes_resources = true
+prow_build_cluster = false
```

Maybe instead of introducing this variable we could check the account ID?

```suggestion
data "aws_caller_identity" "current" {}

locals {
  is_prod = data.aws_caller_identity.current.account_id == "111111111111"
}
```

---

We could, but I don't think that's important because canary will hopefully become a build cluster too.